tests/vcs/git/GitCommitBearTest.py

```diff
@@ -203,7 +217,7 @@ def test_body_checks(self):
 
     def test_different_path(self):
         no_git_dir = mkdtemp()
-        self.git_commit("A shortlog that is too long is not good for history")
+        self.git_commit("Very long shortlog never good for a project history")
```

Comment:
IIUC "long" is actually a verb. A verb may not always be something that
shows action. It could also be a description.  :P
On 5 Apr 2016 01:41, "Zatreanu Adrian-Gabriel" <notifications@github.com>
wrote:

> In tests/vcs/git/GitCommitBearTest.py
> <https://github.com/coala-analyzer/coala-bears/pull/315#discussion_r58439663>
> :
>
> > @@ -203,7 +217,7 @@ def test_body_checks(self):
> >
> >      def test_different_path(self):
> >          no_git_dir = mkdtemp()
> > -        self.git_commit("A shortlog that is too long is not good for history")
> > +        self.git_commit("Very long shortlog never good for a project history")
>
> no verb here either.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/315/files/23d9af66bcb498b19adffe8effb5af1ec8aa4f40#r58439663>
>